apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/ManagedErrorActivity.java

```diff
@@ -72,7 +74,7 @@ protected void send(View view) {
                 Method method = Crashes.class.getDeclaredMethod("trackException", Throwable.class, Map.class);
                 method.setAccessible(true);
                 method.invoke(null, t, properties);
-
+                Iterable<ErrorAttachmentLog> attachmentLogs = AttachmentsUtil.getInstance().getErrorAttachments(getApplicationContext());
```

Comment:
Now we have the code to send attachments, please merge `feature/handled-error-attachments` to your branch and retarget the PR and test in the app.